components/viewer/modal/BackpackModal.vue

```diff
@@ -159,36 +163,33 @@ function copyExportText() {
   }, 4000);
 }
 
-const LEGACY_RX = /^(\w+(\/ON#\d+)?)(,(\w+(\/ON#\d+)?))*$/;
+const IMPORT_RX = /([a-zA-Z0-9-]+)((?:\/ON#|:)(\d*))*/gm;
 function readImportCode() {
   const _code = importCode.value?.trim();
   console.log(`Import Code ${_code}`);
-  if (!_code) return;
+  if (!_code) {
+    console.log(`No import code provided. Clearing Selections...`);
+    selected.value = {};
+    return;
+  }
```

Comment:
This mimics the behavior of MeanDelay's viewer, and importantly, is really convenient if you fuck up the selections array somehow.